php-checks/src/test/resources/checks/UnusedPrivateMethodCheck.php

```diff
@@ -128,3 +128,35 @@ private function appearsInAString() {} // Compliant
 trait SomeTrait {
   private function privateTraitFunction() {}
 }
+
+class DynamicClass
+{
+  /**
+   * @uses one
+   * @uses two()
+   * @uses DynamicClass::three()
+   * @uses DynamicClass::four
+   */
+  public function callDynamic(string $method): void
+  {
+    call_user_func([$this, $method]);
+    // or
+    $this->{$method}();
+  }
+
+  private function one(): void
+  {
+  }
+
+  private function two(): void
+  {
+  }
+
+  private function three(): void
+  {
+  }
+
+  private function four(): void
+  {
+  }
+}
```

Comment:
I would add a private method named `five` (for example) which is not mentioned in a `@uses` and on which we raise an issue. I think that would help ensure that tests on the other methods are not green for a bad reason.